boards/st/nucleo_wl55jc/nucleo_wl55jc.dts

```diff
@@ -115,8 +115,8 @@ stm32_lp_tick_source: &lptim1 {
 	status = "okay";
 };
 
-&i2c2 {
-	pinctrl-0 = <&i2c2_scl_pa12 &i2c2_sda_pa11>;
+&i2c3 {
+	pinctrl-0 = <&i2c3_scl_pb13 &i2c3_sda_pb14>;
 	pinctrl-names = "default";
```

Comment:
That's right on CN8 A4 (PB14) & A5 (PB13) are i2c3 pins
but that's also right that on Arduino connector CN5  D15 (PA12) and D14 (PA11) are i2c2 pins. 
Is it needed to remove i2c2 node ?

Reply:
good point. I can keep its configuration in the devicetree but leave it as `disabled` by default. wdyt?